sprint/sprint-guide.md

```diff
@@ -0,0 +1,50 @@
+(sprint_guide)=
+# Sprint Guide
+
+Here are some guidelines and suggestions about the sprint for participants and volunteers!
+
+## General Information
+
+- Be Polite
+- Ask questions
+- Don't use any discriminatory terms
+- Use gender neutral pronouns unless you know someone's preferred pronouns
+- Don't use words like "guys", instead say "folks" or "people"
+- Be inclusive and kind
+
+The sprint will be happening on Zoom, the sign up instructions are available on Meetup, please try to join on the starting time so you can recieve proper help and guidance for the sprint. There will be breakout rooms inside the Zoom call that participants will be assigned to:
+- Women and Non binary folks: this breakout room is specifically for women and non binary folks, we hope you have a meaningful space here where you're heard!
+- New to open source: this breakout room is for newcomers to open source so you can get suitable help
+- Code related PR: If you are making a code related contribution then you'll be assigned to this breakout room
+- Documentation related PR: If you are making a Documentation related contribution then you'll be assigned to this breakout room
+- Spanish Speaker: If you prefer spanish over english to communicate this breakout room will be useful, Oriol Abril Pla (one of our PyMC volunteers) will be there to help you out
+
+These breakout rooms aren't rigid, you can choose which one you wish to be in, they're created solely to help us help you!
```

Comment:
is there a page on zoom docs on how to do this? we have a use discord 10 min video, we probably need some use zoom reference (no need to make a specific video)